include/boost/http_proto/metadata.hpp

```diff
@@ -54,6 +54,27 @@ enum class payload
     ,to_eof
 };
 
+/** The effective encoding of the body octets.
```

Comment:
Unfortunately the way this is worded makes it unusable to specify going in both directions, which we will want. That's my fault because the text I provided was biased. We need to adjust this documentation to make it clear that it just refers to a particular encoding model and is not biased in direction.